SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
```

Comment:
Название класса намекает на то что он реализует какой то интерфейс сравнения типа таких
![image](https://user-images.githubusercontent.com/6534396/126604243-ce2bfc2d-d866-49e4-8653-bc5deac3bd2a.png)
А по факту нет. Он получает список установленных версий и ищет подходящую версию. Лучше назвать DotNetVersionHelper
```suggestion
    public class DotNetVersionHelper
```